tools/cldr-apps/js/src/views/DashboardWidget.vue

```diff
@@ -74,93 +74,99 @@
         </span>
       </header>
       <section id="DashboardScroller" class="sidebyside-scrollable">
-        <template
-          v-for="entry of data.entries"
-          :key="'template-' + entry.xpstrid"
-        >
-          <template v-if="anyCatIsShown(entry.cats)">
-            <p
-              v-if="!(hideChecked && entry.checked)"
-              :key="'dash-item-' + entry.xpstrid"
-              :id="'dash-item-' + entry.xpstrid"
-              :class="
-                'dash-' + (lastClicked === entry.xpstrid ? ' last-clicked' : '')
-              "
-            >
-              <span class="dashEntry">
-                <a
-                  v-bind:href="getLink(locale, entry)"
-                  @click="() => setLastClicked(entry.xpstrid)"
-                >
-                  <span v-bind:key="cat" v-for="cat of entry.cats">
+        <template v-if="updatingVisibility">
+          <a-spin size="large" />
```

Comment:
```suggestion
          <a-spin delay="1000" size="large" />
```

you can put in a delay so the spinner doesn't show if it would only be there for 1,000 ms (1 second), say.